src/store/migrate.ts

```diff
@@ -104,3 +106,7 @@ export const migrateV7 = (persistedState: LocalStorageInterfaceV7oV8) => {
     chat.id = uuidv4();
   });
 };
+
+export const migrateV8 = (persistedState: LocalStorageInterfaceV8toV9) => {
+  persistedState.responseFormat = defaultResponseFormat;
+}
```

Comment:
provided you got it saving similar to other props (e.g. max_tokens) this should work
```tsx
export const migrateV8 = (persistedState: LocalStorageInterfaceV8toV9) => {
 persistedState.chats.forEach((chat) => {
    chat.config = {
      ...chat.config,
      responseFormat : defaultResponseFormat,
    };
  });
}
```

I would rename `responseFormat` to `response_format` for consistency purposes